pennylane/_device.py

```diff
@@ -183,7 +184,7 @@ def capabilities(cls):
         """
         return cls._capabilities
 
-    def execute(self, queue, observables):
+    def execute(self, queue, observables, parameters=None):
```

Comment:
I would either make this positional or `parameters={}` which is the default when there are no parameters, isn't it?

Reply:
I suppose best to make it positional, since it will be passed by the QNode every time under this change